src/jest/custom-matchers.ts

```diff
@@ -17,8 +17,9 @@ export const customTestMatchers = {
     let expectedMarble: string;
 
     if (haveValueObjects(actual, expected)) {
-      actualMarble = JSON.stringify(actual);
-      expectedMarble = JSON.stringify(expected);
+      const spaces = 2;
+      actualMarble = JSON.stringify(actual, null, spaces);
+      expectedMarble = JSON.stringify(expected, null, spaces);
```

Comment:
Typescript has the type set to `string | number`, for the spacing wanted

Reply:
It seems all right, what is the problem?

Reply:
oh, no problem at all! was just adding the comment as you previously mentioned `JSON.stringify(actual, null, true)`. sorry if that caused alarm haha

Reply:
Right, it's a number of spaces, my mistake )